internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site-manifest.yml

```diff
@@ -1,16 +1,16 @@
-# The manifest for the "{{.Name}}" service.
-# Read the full specification for the "{{.Type}}" type at:
+# The manifest for the "static" service.
+# Read the full specification for the "Static Site" type at:
 #  https://aws.github.io/copilot-cli/docs/manifest/static-site/
 
 # Your service name will be used in naming your resources like S3 buckets, etc.
 name: static
 type: Static Site
 
 files:
-  - context: ./frontend
-    source: dist
+  - source: ./frontend/dist
     destination: /dist
   - source: ./foo 
+    recursive: true
     destination: /code 
-    include: "*.jpg"
-    exclude: "*"
\ No newline at end of file
+    exclude: "*"
+    reinclude: "*.jpg"
```

Comment:
qq: which part of the code change in this PR lead to this change in `testdata` ?

Reply:
The `yaml` field and template always had `reinclude`, actually. It was just wrong here in the `testdata`.

Reply:
aye then I wonder why the test wasn't failing before :panic